packages/app/src/cli/services/dev.ts

```diff
@@ -107,6 +114,11 @@ async function dev(options: DevOptions) {
     )
     proxyTargets.push(devExt)
   }
+  if (options.app.extensions.theme.length > 0) {
+    const extension = options.app.extensions.theme[0]!
+    const devExt = await devThemeExtensionTarget(options, extension, apiKey, adminSession!, storefrontToken!, token)
+    proxyTargets.push(devExt)
```

Comment:
The theme extension doesn't need to be accessed through the proxy, right?
(i.e., doesn't need to be access publicly via ngrok for instance)

Then instead of adding it to `proxyTargets`, you should add it to `additionalProcesses`, like we do with the backend (that is not accessible directly through the tunnel)

⚠️  A note for the future: we could have a public URL so you can share it, something like:
`https://{tunnel_url}/theme-app-ext`. In that case then it should be added to `proxyTargets` and we'd need to update the proxy

Reply:
Precisely, the theme extension doesn't need to be accessed through the proxy indeed. I've applied the fix - Thank you! :)